src/Compilers/VisualBasic/Portable/BoundTree/BoundNodes.xml

```diff
@@ -642,6 +642,8 @@
       
         <Field Name="ReceiverOpt" Type="BoundExpression" Null="allow"/>
         <Field Name="Arguments" Type="ImmutableArray(Of BoundExpression)"/>
+        <!-- Which arguments are default. Used by IOperation for determining ArgumentKind. -->
+        <Field Name="DefaultArguments" Type="BitVector" Null="NotApplicable" />
```

Comment:
>BitVector [](start = 45, length = 9)

We list all the value types used at the beginning of the root node. I do not remember exact details, but that probably affects how the classes are generated

Reply:
Ah, I missed that. When I was looking at how ValueTypes were generated I modified the generator, I'll take this approach instead.

---
In reply to: [190011872](https://github.com/dotnet/roslyn/pull/26781#discussion_r190011872) [](ancestors = 190011872)